charts/azad-kube-proxy/values.yaml

```diff
@@ -37,7 +37,7 @@ secret:
   CLIENT_SECRET: ""
   TENANT_ID: ""
 
-replicaCount: 1
+replicaCount: 2
```

Comment:
Most testing has been done with one replica so far. Is it necessary to have two right now?

I would also consider using redis if we are using two replicas, making sure we don't have to lookup each token twice as well as sync the groups twice.

One thing that may be worth doing is configuring the persistence / stickiness to make sure we send the same client to the same pod when using memory cache and multiple replicas, to keep the MS Graph requests to a minimum as well as the end user experience as good as possible.